### PR TITLE
Tests for /examples

### PR DESCRIPTION
**Pull request by 650elx**

### CODE REVIEW

File: tests/examples/test_flask_example.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+):
+    sinch_client.configuration.auth_origin = auth_origin
+    sinch_client.configuration.conversation_origin = conversation_origin
+    sinch_client.configuration.disable_https = True
+    app.testing = True
+    flask_client = app.test_client()
+
+    response = flask_client.post("/create_app")
+    assert response.status_code== 200
+    assert "sinch_app_id" in response.json
```
Comment (asein-sinch) [inline tests/examples/test_flask_example.py:16]: What happens if `sinch_client.conversation.app.create()` fails? Will `sinch_app_id` contain an empty value? In this case, will this test succeed or fail?

File: tests/examples/test_flask_example.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    conversation_origin
+):
+    sinch_client.configuration.auth_origin = auth_origin
+    sinch_client.configuration.conversation_origin = conversation_origin
+    sinch_client.configuration.disable_https = True
+    app.testing = True
+    flask_client = app.test_client()
+
+    response = flask_client.post("/create_app")
+    assert response.status_code== 200
```
Comment (Dovchik) [inline tests/examples/test_flask_example.py:15]:
```suggestion
    assert response.status_code == 200
```
Autoformatter?

**Review by Dovchik — APPROVED** [review]
LGTM modulo Antoine comments and formatter

File: tests/examples/test_flask_example.py (hunk 2 above)
Comment (650elx) [reply tests/examples/test_flask_example.py:15]: Something is wrong with flake8 (CI) settings. I'll fix that.

File: tests/examples/test_flask_example.py (hunk 1 above)
Comment (650elx) [reply tests/examples/test_flask_example.py:16]: This is just a simple integration test - I want to know if examples are working correctly against current version of the SDK. When `sinch_client.conversation.app.create()` fails, flask returns HTTP 500, hence the test fails.

**Review by JPPortier — APPROVED** [review]

**Pull request merged**